WooCommerce/Classes/POS/Presentation/TotalsView.swift

```diff
@@ -130,6 +130,15 @@ private extension TotalsView {
                               shimmeringActive: totalsLoading,
                               matchedGeometryId: Constants.matchedGeometrySubtotalId)
             Spacer().frame(height: Constants.subtotalsVerticalSpacing)
+
+            if viewHelper.shouldShowTotalDiscountField(cart: posModel.cart, orderTotals: orderTotals) {
```

Comment:
Just wondering: 

We seem to create new instances of `TotalsViewHelper` and/or reference the existing one across the files indiscriminately, and that's fine as it's just stateless and basically functional: Should we make this object just a struct and call static functions? That should add more consistency of how we use it and clarify we don't need to retain a certain state for it to work.

Reply:
I agree. Our intention is to use these helpers as esentially View Models with immutable helper methods.

I think the best approach is to turn helpers into structs. If someone wants to make it stateful, they would need to declare mutating funcs and they would get compiler errors within the views with:

> Cannot use mutating member on immutable value: 'self' is immutable